packages/serverless/test/backend_connector_test.js

```diff
@@ -0,0 +1,219 @@
+/*
+ * (c) Copyright IBM Corp. 2022
+ */
+
+'use strict';
+
+const expect = require('chai').expect;
+const sinon = require('sinon');
+const backendConnector = require('../src/backend_connector');
+const uninstrumentedHttp = require('../src/uninstrumentedHttp');
+
+const sleep = async ts => {
```

Comment:
_suggestion_: We already have a util for this, that we could use here: https://github.com/instana/nodejs/blob/main/packages/core/test/test_util/delay.js